google/resource_google_project.go

```diff
@@ -310,3 +336,9 @@ func resourceGoogleProjectDelete(d *schema.ResourceData, meta interface{}) error
 	d.SetId("")
 	return nil
 }
+
+func resourceProjectImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	// Explicitly set to default as a workaround for `ImportStateVerify` tests.
```

Comment:
It is not only for ImportStateVerify but also to ensure people don't see a diff after importing this resource.

Reply:
Great point, I'll modify that.